website/api/controllers/customers/save-billing-info-and-subscribe.js

```diff
@@ -82,6 +97,9 @@ module.exports = {
       billingCardLast4: inputs.paymentSource.billingCardLast4,
       billingCardExpMonth: inputs.paymentSource.billingCardExpMonth,
       billingCardExpYear: inputs.paymentSource.billingCardExpYear,
+      firstName: inputs.firstName ? inputs.firstName : this.req.me.firstName,
+      lastName: inputs.lastName ? inputs.lastName : this.req.me.lastName,
+      organization: inputs.organization ? inputs.organization : this.req.me.organization
```

Comment:
You don't need the ternary operators here.

If these are omitted from the request, then they will come through as `undefined`.  To an `.update()` (or any waterline method), a key with an `undefined` value is the same thing as excluding the key.  (i.e. we equate the concept of `foo.bar === undefined` and `!foo.hasOwnProperty('bar')`.  Because it's silly to have to keep track of the difference.)

In other words:

```
  firstName: inputs.firstName,
  lastName: inputs.lastName,
  organization: inputs.organization,
```

Risk: If the frontend code includes these values as empty string (`''`), then they will clear out whatever was there before.  So be sure that's not happening.